backend/routing/daemon_controller.py

```diff
@@ -149,7 +155,53 @@ async def force_downgrade(
             logger=request.logger,
             regenerate_from=regenerate_from,
         )
-                
+
```

Comment:
This func was deleted on purpose, since adding documents is done by popping the fileid onto the processing queue, it shouldnt be getting called by anything